internal/network/manifests_generator.go

```diff
@@ -400,6 +411,12 @@ func createDnsmasqForSingleNode(log logrus.FieldLogger, cluster *common.Cluster)
 		"UNMANAGED_RESOLV_CONF": base64.StdEncoding.EncodeToString([]byte(unmanagedResolvConf)),
 	}
 
+	// Default is less than 4.13, no need to check error, if version not set no need to add additional params
+	rhel9Version, _ := common.VersionGreaterOrEqual(cluster.OpenshiftVersion, "4.13.0-0.0")
```

Comment:
just wanted to make sure (you can also add unit-tests for those cases),
it works for both pre-release 4.13 versions (which I see you tested) and ``4.13.0``?

Reply:
you mean to test version greater? To say the truth i assumed it was tested already and not require retesting each time we use it. 
As far as i understand 0-0.0 means the minimal 4.13 and any version of 4.13 will hit it